### ISSUE TITLE
Mejores notificaciones

### ISSUE THREAD

**Issue by gpascualg** [Push-EDX/flarum-ext-chat#14]
Las notificaciones ahora mismo son muy simples, se podrían mejorar con:

- [ ] Incluir el autor en el título
- [ ] Pequeña imagen del foro o parecido

https://github.com/Push-EDX/flarum-ext-chat/blob/master/js/forum/src/components/ChatFrame.js#L326

Como punto extra, se podría unificar

* https://github.com/Push-EDX/flarum-ext-chat/blob/master/js/forum/src/components/ChatFrame.js#L326
* https://github.com/Push-EDX/flarum-ext-chat/blob/master/js/forum/src/components/ChatFrame.js#L332

Que es exactamente lo mismo

**Comment by gpascualg** [comment]
Not working? I don't get them to show. Can someone reproduce?

**Comment by jtojnar** [comment]
@gpascualg We cannot get them to show either.

**Comment by Telematica** [comment]
Notification are silent by default, according the JS code here:
https://github.com/Push-EDX/flarum-ext-chat/blob/63847ba0286142611816f7f1efda17c8b08b8be4/js/forum/src/components/ChatFrame.js#L196

I guess we can delegate enabling notifications to the Extension settings on the Admin dashboard... thoughts on this @gpascualg?

**Comment by gpascualg** [comment]
@Telematica Indeed, default behaviour is muted. We might have an option on the Admin dashboard to change the defaults, it would be a nice addition.

The problem here, however, is that it seems that whether user-enabled or not, the sound doesn't play. Please bear in mind this code (specially notifications/audio) is screaming "refactor me!"...

**Comment by Telematica** [comment]
@gpascualg I'm interested to be involved on the Flarum development, and its community... I'm starting a new project, and I'm using Flarum as an alternative, hope it soon could be released as stable project (at least 1st version). I will not hesitate to perform a pull request on this. ty!

**Comment by gpascualg** [comment]
Nice @Telematica, any PR would be much appreciated, I'm really busy right now and can't throw much time into this :(

**Comment by gpascualg** [comment]
Ah... I've just reviewed the code and you were absolutely right! Regardless of user settings I am always passing `false` to `notify`.

We should modify
https://github.com/Push-EDX/flarum-ext-chat/blob/63847ba0286142611816f7f1efda17c8b08b8be4/js/forum/src/main.js#L15
To trigger notifications with this particular data (as they are new messages)

and 
https://github.com/Push-EDX/flarum-ext-chat/blob/63847ba0286142611816f7f1efda17c8b08b8be4/js/forum/src/main.js#L36
To disable them (old missatges fetched at start)

Finally,
https://github.com/Push-EDX/flarum-ext-chat/blob/63847ba0286142611816f7f1efda17c8b08b8be4/js/forum/src/components/ChatFrame.js#L251
would change notify based on those

**Comment by Telematica** [comment]
@gpascualg I'll be issuing a PR soon to fix notification and do the dashboard notification delegation stuff, so you can review it and merge if you think is OK.

**Comment by gpascualg** [comment]
@Telematica Any news on the PR?

**Comment by Telematica** [comment]
Sorry, not yet, I've been a bit busy. But a new project is coming; in which I'll need to plug this ext to a plain Flarum instance, so I'll got a chance to make refactors and fixes for this. I'll keep you posted.

**Comment by gpascualg** [comment]
It's okay! Keep an eye to the issue though, if I grasp a bit of time I might fix it myself, in which case I would close the issue
